lib/kino_flame/runner_cell.ex

```diff
@@ -5,13 +5,33 @@ defmodule KinoFLAME.RunnerCell do
   use Kino.JS.Live
   use Kino.SmartCell, name: "FLAME runner cell"
 
-  @text_fields ["name"]
+  @text_fields ["name", "backend"]
   @number_fields ["min", "max", "max_concurrency"] ++ ["fly_cpus", "fly_memory_gb", "fly_gpus"]
+  @default_pod_template """
+  apiVersion: v1
+  kind: Pod
+  metadata:
+    generateName: livebook-flame-runner-
+  spec:
+    containers:
+      - image: ghcr.io/livebook-dev/livebook:nightly
+        name: livebook-runtime
+        resources:
+          limits:
+            cpu: "1"
+            memory: 1Gi
+          requests:
+            cpu: "1"
+            memory: 1Gi
+  """
 
   @impl true
   def init(attrs, ctx) do
+    backend = attrs["backend"] || ((System.get_env("KUBERNETES_SERVICE_HOST") && "k8s") || "fly")
```

Comment:
Let's move this to a function:

```suggestion
    backend = attrs["backend"] || default_backend()
```